packages/@okta/vuepress-site/docs/reference/api/devices/index.md

```diff
@@ -0,0 +1,855 @@
+---
+title: Devices
+category: management
+---
+
+# Device API
+
+<ApiLifecycle access="ie" /><br>
+<ApiLifecycle access="Limited GA" /><br>
+
+> **Note:** This feature is only available as a part of the Okta Identity Engine. Please [contact support](mailto:dev-inquiries@okta.com) for further information.
```

Comment:
```suggestion
> **Note:** This feature is only available as a part of Okta Identity Engine. Please [contact support](mailto:dev-inquiries@okta.com) for further information.
```

Reply:
Can we please update https://oktawiki.atlassian.net/wiki/spaces/DOC/pages/2004649380/OIE+API+Reference+Documentation+Guidance?atlOrigin=eyJpIjoiOGM5MGM4Y2RlNDczNDNmNjg3NmJjZDBhN2JmZDFjM2MiLCJwIjoiY29uZmx1ZW5jZS1jaGF0cy1pbnQifQ as I copy + pasted the text